rest/model/test_results.go

```diff
@@ -28,7 +31,7 @@ func (a *APITestResult) Import(i interface{}) error {
 		a.TestName = ToAPIString(tr.TestName)
 		a.Trial = tr.Trial
 		a.Status = ToAPIString(tr.Status)
-		a.LogURL = ToAPIString(tr.LogURL)
+		a.LogURL = ToAPIString(strings.Join([]string{"test_log", tr.TaskID, strconv.Itoa(tr.Execution), tr.TestName}, "/"))
```

Comment:
This URL should be: `<cedar_host>/rest/v1/buildlogger/test_name/{task_id}/{test_name}?execution={execution}`.

Reply:
Fixed.